### ISSUE TITLE
Ask for schedule

### ISSUE THREAD

**Issue by jbburns** [evanpetzoldt/qsignups-lambda#25]
Would be nice to be in channel #ao and ask
"@QSignsups, who has the Q?"

And it responds to the channel with that channel's schedule for the week

If inside a channel that it doesn't recognize, maybe it could ask what AO or if you want the entire region

**Comment by evanpetzoldt** [comment]
yeah, we've talked about this! There's some fun stuff we could do with this (randomly pick a Q, etc). If you haven't gotten to this yet, I do have weekly "Q Lineups" that are sent out to each channel. Something like this could fill the gap if that gets buried though.
